workers/unity/Packages/com.improbable.gdk.tools/Common.cs

```diff
@@ -95,7 +95,12 @@ public static string GetPackagePath(string packageName)
             try
             {
                 var package = Json.Deserialize(File.ReadAllText(filePath, Encoding.UTF8));
-                return ((Dictionary<string, object>) package["dependencies"]).ToDictionary(kv => kv.Key,
+                if (!package.TryGetValue("dependencies", out var dependenciesJson))
```

Comment:
If you replace `var` with `Dictionary<string, object>`, we might be able to do without the cast below?

Reply:
You can't do the cast in the out parameter declaration sadly